src/config/tags.ts

```diff
@@ -21,6 +21,11 @@ export const tags: Array<Tag> = [
     message:
       "Don't ask to ask, just ask\n<https://dontasktoask.com>. For details on how to ask a strong question, visit https://canary.discord.com/channels/692816967895220344/1033845883822948502",
     name: "How to ask a question",
+  },
+    {
+    message:
+      "Please email donors@freecodecamp.org and provide the email you use to log into fCC and your donation receipt.",
```

Comment:
```suggestion
      "Please email donors@freecodecamp.org and provide the email you use to log into freecodecamp.org/learn, your donation receipt, and a brief explanation of your issue.",
```

- So they don't just send the receipt with no explanation
- So they use their learn email, which may be different from their forum email (or even Discord, I suppose someone could confuse that too)